### ISSUE TITLE
check Printf/Errorf argument count against format

### ISSUE THREAD

**Issue by zeisss** [golang/lint#188]
I would have expected golint to "throw" an error here.

```
$ golint test.go
$ cat test.go
package main

import "fmt"

func main() {
   fmt.Printf("Hello %s!\n")
}
```

**Comment by dsymonds** [comment]
That's something that govet checks. Try running `go vet test.go`.
